frontend/views/pages/GroupSettings.vue

```diff
@@ -0,0 +1,138 @@
+<template lang="pug">
+page(pageTestName='dashboard' pageTestHeaderName='groupName' v-if='currentGroupState')
+  template(#title='') Group Settings
+
+  page-section(title='')
+    i18n(tag='p') Changes to these settings will be visible to all group members
+
+    form
+      .field
+        i18n.label(tag='label') Group name
+
+        input.input.is-large.is-primary(
+          ref='name'
+          type='text'
+          name='groupName'
+          :class="{ 'error': $v.form.groupName.$error }"
+          :value='currentGroupState.groupName'
+          @input='update'
+          @keyup.enter='next'
+          data-test='groupName'
+        )
+
+      .field
+        i18n.label(tag='label') Description
+
+        textarea.textarea(
+          name='sharedValues'
+          ref='purpose'
+          placeholder='Group Purpose'
+          maxlength='500'
+          :class="{ 'error': $v.form.sharedValues.$error }"
+          :value='currentGroupState.sharedValues'
+          @input='update'
+        )
+
+      .field
+        i18n.label(tag='label') Default currency
+        .select-wrapper
+          select(
+            name='incomeCurrency'
+            required=''
+            :value='currentGroupState.incomeCurrency'
+            @input='update'
+          )
+            option(
+              v-for='(symbol, code) in currencies'
+              :value='code'
+              :key='code'
+            ) {{ symbol }}
+
+        i18n(tag='p') This is the currency that will be displayed for every member of the group, across the platform.
+
+      i18n.is-success(
+        tag='button'
+        ref='save'
+        @click='save'
+        :disabled='$v.form.$invalid'
+        data-test='saveBtn'
+      ) Save changes
+
+  page-section(title='Leave Group')
+    i18n(tag='p') This means you will stop having access to the #[b group chat] (including direct messages to other group members) and #[b contributions]. Re-joining the group is possible, but requires other members to vote and reach an agreement.
```

Comment:
For some reason, it seems the interpolation of <b> isn't working correctly. Do you know why? Based on [Pug Docs](https://pugjs.org/language/interpolation.html) it seems fine. Maybe we need something else... 🤔 

![image](https://user-images.githubusercontent.com/14869087/62692977-83a98200-b9c9-11e9-9509-001cfc0828ec.png)

Reply:
@sandrina-p in situations where interpolation is needed, if the pug dependency is broken, then we can use the `L` function instead (along with normal HTML markup).

Reply:
Hum.. I'm not understanding. can you be more explicit in how exactly should it be?

Reply:
@sandrina-p from: https://okturtles.slack.com/archives/C0EH7P20Y/p1564533561151800

> and if that too doesn't work, then yeah, maybe in that situation use the `L` function and pass in HTML like so: `{{ L('<b>foo</b>') }}` or whatever is the syntax

Reply:
got it, thanks!